src/main/java/com/kodilla/ecommercee/controller/user/UserController.java

```diff
@@ -0,0 +1,26 @@
+package com.kodilla.ecommercee.controller.user;
+
+import com.kodilla.ecommercee.domain.user.UserEntityDto;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/v1/user")
+public class UserController {
+
+    @RequestMapping(method = RequestMethod.POST, value = "createUser", consumes = MediaType.APPLICATION_JSON_VALUE)
+    public void createUser(@RequestBody UserEntityDto userEntityDto){
+
+    }
+
+    @RequestMapping(method = RequestMethod.PUT, value = "blockUser", consumes = MediaType.APPLICATION_JSON_VALUE)
+    public UserEntityDto blockUser(@RequestParam String login) throws UserNotFoundException{
+        return new UserEntityDto(1L, "John", "Rambo", "JohnnyR",
+                "theOne&OnlyJohnRambo@disaster.boom", "bullet", 12345, true );
```

Comment:
jak blokujemy usera to dałbym na końcu false - powinniśmy go wylogować

Reply:
To są dummy data moge zmienić jeśli chcesz ale nie widzę potrzeby.